scripts/zkai_d128_proof_artifact_backend_spike_gate.py

```
@@ -964,6 +1141,58 @@ def validate_payload(payload: Any, *, require_mutations: bool = True) -> None:
+        bridge_probe.get("status"),
+        "GO_D128_RMSNORM_TO_PROJECTION_BRIDGE_ONLY",
+        "d128 RMSNorm-to-projection bridge status",
+    )
+    expect_equal(
+        bridge_probe.get("present_symbols"),
+        list(D128_BRIDGE_SYMBOLS),
+        "present d128 RMSNorm-to-projection bridge symbol inventory",
+    )
+    expect_equal(
+        bridge_probe.get("projection_input_relabels_full_output"),
+        False,
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)